tests/e2e/hooks.ts

```diff
@@ -0,0 +1,29 @@
+import { Before, BeforeAll, After, AfterAll, setDefaultTimeout } from '@cucumber/cucumber'
+import { chromium } from '@playwright/test'
+import config from './config'
+import { deleteAllFiles, emptyTrashbin } from './utils/helpers'
+
+setDefaultTimeout(config.timeout * 1000)
+
+BeforeAll(async function (): Promise<void> {
+  global.browser = await chromium.launch({
+    slowMo: config.slowMo,
+    headless: config.headless,
+    channel: 'chrome'
+  })
+})
+
+Before(async function (): Promise<void> {
+  global.context = await global.browser.newContext({ ignoreHTTPSErrors: true })
+  global.page = await global.context.newPage()
+})
+
+AfterAll(async function (): Promise<void> {
+  await global.browser.close()
+})
+
+After(async function (): Promise<void> {
+  await deleteAllFiles()
+  await emptyTrashbin()
+  await global.page.close()
+})
```

Comment:
@saw-jan would you like report tracing to be added to your extension? 
It seems like currently we're not doing this in any of "our" ocis web extensions... like mentioned [above](https://github.com/saw-jan/web-app-3dmodel-viewer/pull/12#discussion_r1606427095), I think it would be good to keep all the extensions we have influence on as much aligned in these matters as possible

Reply:
opened an issue for Amrita's suggestion about report tracing, this improvement will be addressed in a separate PR, see https://github.com/saw-jan/web-app-3dmodel-viewer/issues/17